src/index.js

```diff
@@ -0,0 +1,104 @@
+import { handleDragStart, handleDrop, cancelDefault } from './services/sort';
+import {
+  updateStatus, getTodos, saveToStorage, orderAndSave,
+} from './services/update';
+import {
+  createTodo, clearAll, clearList, editTodoInStorage,
+  editTodoInList, deleteTodoFromStorage, deleteTodoFromList,
+} from './services/todoService';
+import './style.css';
+
+class TodoList {
+  constructor() {
+    this.todos = getTodos();
+  }
+
+ handleCompletedStatus = (e) => {
+   const todos = getTodos();
+   updateStatus(e, todos);
+ }
+
+ handleEditTodo = (e) => {
+   if (e.keyCode === 13) {
+     const { value } = e.target;
+     const index = parseInt(e.target.dataset.index, 10);
+     editTodoInStorage(value, index);
+     editTodoInList(value, index);
+   }
+ }
+
+ handleDeleteTodo = (e) => {
+   const index = parseInt(e.target.dataset.index, 10);
+   deleteTodoFromList(index);
+   deleteTodoFromStorage(index);
+ }
+
+ handleClearAllCompleted = () => {
+   const todos = getTodos();
+   const filteredTodos = clearAll(todos);
+   orderAndSave(filteredTodos);
+   clearList();
+   this.displayList();
+ }
+
+ handleNewTodo = (e) => {
+   const todos = getTodos();
+   if (e.keyCode === 13) {
+     const newTodo = createTodo(e.target.value, todos);
+     e.target.value = '';
+     todos.push(newTodo);
+     clearList();
+     saveToStorage(todos);
+     this.displayList();
+   }
+ }
+
+ displayList() {
+   const container = document.querySelector('.todoList');
+   getTodos().forEach((todo, index) => {
+     const li = document.createElement('li');
+     li.innerHTML = `
+                  <div class="task-item">
+                    <div class="task-info">
```

Comment:
* I think the property index can be set to the value of the array length  (i.e. if you're adding a 5th task to the list, the index of that task should equal 5) [Please refer to the project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=By%20default%20new%20tasks%20should%20have%20the%20property%20completed%20set%20to%20false%20and%20the%20property%20index%20set%20to%20the%20value%20of%20the%20new%20array%20length%20(i.e.%20if%20you%27re%20adding%20a%205th%20task%20to%20the%20list%2C%20the%20index%20of%20that%20task%20should%20equal%20to%205).) 
* HINT 💭 You can set your index by doing (index: x.length + 1 ) so that the first item created don't take the index `0`